sdk/src/wallet/utxo.ts

```diff
@@ -69,20 +70,28 @@ export async function createBitcoinPsbt(
     const addressInfo = getAddressInfo(fromAddress);
 
     // TODO: possibly, allow other strategies to be passed to this function
-    const utxoSelectionStrategy = 'default';
+    let utxoSelectionStrategy = 'default';
+    if (addressInfo.type === AddressType.p2tr) {
+        utxoSelectionStrategy = 'all';
```

Comment:
with `default` strategy I was not able to create psbt
also tests inside @scure/btc-signer use `all` as a strategy for `selectUTXO`

Reply:
Why not? Would be good to know why

Reply:
I will get into why it's causing in issue. Will update